pkg/pillar/cmd/zedrouter/flowstats.go

```diff
@@ -660,6 +657,10 @@ func dnsDataRemove(bnNum int) {
 
 func bridgeStrToNum(bnStr string) (int, error) {
 	bnNumStr := strings.Split(bnStr, "bn")
+	if len(bnNumStr) < 2 {
```

Comment:
What if len is > 2?
Seems better to just use strings.TrimSuffix(bnStr, "bn), which always returns a string, and then feed that into Atoi. The latter will fail if it isn't "bn%d"

Reply:
sure.